tests/manager/celery/tasks/test_lexile.py

```
@@ -0,0 +1,485 @@
+            )
+
+        replace_calls: list[tuple] = []
+
+        def capture_replace(*args: object, **kwargs: object) -> None:
+            replace_calls.append((args, kwargs))
+            raise RuntimeError("Replace captured (avoid actual replacement)")
+
+        with patch.object(
+            lexile.lexile_db_update_task, "replace", side_effect=capture_replace
+        ):
+            with pytest.raises(RuntimeError, match="Replace captured"):
```

Ruff (0.15.7)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)